test/test.js

```diff
@@ -210,13 +210,19 @@ ruleTester.run('generated-flow-types', rules['generated-flow-types'], {
     {code: 'graphql`query {{{`'},
     {
       code: `
-        import type {TestFragment_foo} from 'TestFragment_foo.graphql';
+        import type {TestFragment_foo$ref} from 'TestFragment_foo.graphql';
         useFragment<FooResponse>(graphql\`query TestFragment_foo { id }\`)
```

Comment:
what's the purpose of having this `<FooResponse>` here?

Reply:
Copypaste from `useQuery`, removing.